frontend/src/pages/server/Administration.tsx

```diff
@@ -127,9 +127,49 @@ export function Administration({setServ, serv, onServerFilesDeleted}: Props) {
                 </div>
             </Card>
             <Card variant="soft"  className={''}>
-                <Typography level="title-md">
-                    Server startup
-                </Typography>
+                <div className={'space-x-4 w-full flex justify-between'}>
+                    <Typography level="title-md">
+                        Server startup
+                    </Typography>
+                    <Typography level="title-md">
+                        <div className={'space-x-4 w-full flex'}>
+                            <div className={'inline-block'}>
+                                <Modal open={showServerCommandModalOpen} onClose={() => setShowServerCommandModalOpen(false)}>
+                                    <ModalDialog variant="outlined" role="dialog">
+                                        <DialogTitle>
+                                            <IconInfoCircle/>
+                                            Server startup command
+                                        </DialogTitle>
+                                        <Divider />
+                                        <DialogContent>
+                                            {serverCommand}
+                                        </DialogContent>
+                                        <DialogActions>
+                                            <Button variant="solid" color="neutral" onClick={() => setShowServerCommandModalOpen(false)}>
+                                                Close
+                                            </Button>
+                                        </DialogActions>
+                                    </ModalDialog>
+                                </Modal>
+                                <Button color='neutral' onClick={() => {
+                                    setShowServerCommandModalOpen(true)
+                                    GetServerWithError(serv.id, false).then(() => {
```

Comment:
you are not actually using the retunred server, so you can remove GetServerWithError